roofit/roofitmore/src/RooHypatia2.cxx

```diff
@@ -119,21 +119,21 @@
 /// \param[in] a2 Start of right tail.
 /// \param[in] n2 Shape parameter of right tail (\f$ n2 \ge 0 \f$). With \f$ n2 = 0 \f$, the function is constant.
 RooHypatia2::RooHypatia2(const char *name, const char *title, RooAbsReal& x, RooAbsReal& lambda,
-    RooAbsReal& zeta, RooAbsReal& beta, RooAbsReal& sigm, RooAbsReal& mu, RooAbsReal& a,
+    RooAbsReal& zeta, RooAbsReal& beta, RooAbsReal& sigma, RooAbsReal& mu, RooAbsReal& a,
```

Comment:
It was `sigm` on purpose. Otherwise, the ancient compilers that think you cannot name an argument like a function issue a shadow warning. Maybe it should be called `_sigma`, so it doesn't look like a typo.
In the header, the name is correct, btw.

Reply:
@hageboeck can we merge this as is, and fix this specific case in a second PR, please?

I'd like to move this in soon, else we run the risk of creating conflicts as the PR is quite large. And I really don't want to ask Matthias to spend even more time on his...

Reply:
Hotfixed by directly editing the file on github. I guess you can merge as soon as `ROOT-performance-centos7-multicore/default` completes without issuing a shadow warning.